docs/conf.py

```diff
@@ -77,3 +96,15 @@
      'examples_dirs': '../examples',
      'gallery_dirs': 'auto_examples',
 }
+
+# -- LaTeX macros ------------------------------------------------------------
+
+mathjax_config = {                  
+    "TeX": {                        
+        "Macros": {                 
+            "E": '{\\mathbb{E}}',
```

Comment:
We're going to need to tell users where to find these macros.

Reply:
(not necessarily in this PR)

Reply:
Not sure I understand. Why would users have to know about logic to generate a webpage that they're reading?

Reply:
s/users/contributors/

Reply:
I see! It says "LaTeX macros" right above. That's what I would search for. I do agree that we need some sort of "readme" for the doc generation, though. I'll start with that, and we can keep adding details over the next few PRs.

Reply:
Yes... having something you have to find using `grep -ilr` is not ideal :-)

Reply:
I've added a README. I still think people either know how sphinx works in which case they'll go to conf.py, or they don't, in which case they'll search for LaTeX and will find it. Anyway, it's documented now.